contributing-guides/translation-templates/subcommand-mention.md

```diff
@@ -72,7 +72,9 @@ Not translated yet.
 ---
 ### fr
 
-Not translated yet.
+```markdown
+Certaines commandes comme `example command` ont leur propre documentation.
```

Comment:
Is there a maybe a more accurate translation of `sub-commands`? How about `sous-commandes`?